src/main/java/news/caughtup/model/Article.java

```diff
@@ -19,18 +23,29 @@ public String getTitle() {
     public void setTitle(String title) {
         this.title = title;
     }
-    public String getDate() {
+    
+    public int getResourceId() {
+        return resourceId;
+    }
+
+    public void setResourceId(int resourceId) {
```

Comment:
yeah because if we read from DB we set the resource_id from the constructor and that shouldn't change in runtime.